data/core/config.lua

```diff
@@ -47,7 +47,7 @@ config.plugins = {}
 -- Allow you to set plugin configs even if we haven't seen the plugin before.
 setmetatable(config.plugins, {
   __index = function(t, k)
-    if rawget(t, k) == nil then rawset(t, k, {}) end
+    if rawget(t, k) == true or rawget(t, k) == nil then rawset(t, k, {}) end
```

Comment:
Maybe the return value of `rawget(t, k)` should be stored on a variable so the lookup is performed just once to check if true or nil

Reply:
Sure; good call.